File: Beginner/1048 (Salary Increase)/1048.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            double x = double.Parse(Console.ReadLine());
            double percentual = 0.0;
            double reajuste = 0.0;
            double soma = 0.0;
         

            if(x > 0 && x<= 400.00)
            {
                percentual = 15;
                reajuste = x * (percentual / 100);
                soma = x + reajuste;
            }
            else if(x > 400.00 && x <= 800.00)
            {
                percentual = 12;
                reajuste = x * (percentual / 100);
                soma = x + reajuste;
            }
            else if (x > 800.00 && x <= 1200.00)
            {
                percentual = 10;
                reajuste = x * (percentual / 100);
                soma = x + reajuste;
            }
            else if (x > 1200.00 && x <= 2000.00)
            {
                percentual = 7;
                reajuste = x * (percentual / 100);
                soma = x + reajuste;
            }
            else if (x > 2000.00)
            {
                percentual = 4;
                reajuste = x * (percentual / 100);
                soma = x + reajuste;
            }

            Console.WriteLine("Novo salario: " + string.Format("{0:0.00}",soma));
            Console.WriteLine("Reajuste ganho: " + string.Format("{0:0.00}", reajuste));
            Console.WriteLine("Em percentual: " + string.Format("{0:0}", percentual) + " %");

        }
    }
}

File: Beginner/1048 (Salary Increase)/1048.cpp
#include <iostream>
using namespace std;

int main(){
  float x=0,soma=0,rej=0,percent=0;
  cin >> x;

  if(x>0 && x<=400.00){
      percent = 15;
      rej=x*(percent/100);
      soma = x + rej;
  }
  else if (x>400.00 && x<=800.00){
      percent = 12;
      rej=x*(percent/100);
      soma = x + rej;
  }
  else if (x>800.00 && x <= 1200.00){
      percent = 10;
      rej=x*(percent/100);
      soma = x + rej;
  }
  else if (x >1200.00 && x <=2000.00){
      percent = 7;
      rej=x*(percent/100);
      soma = x + rej;
  }
  else if (x>2000.00){
      percent = 4;
      rej=x*(percent/100);
      soma = x + rej;
  }

    printf("Novo salario: %.2f\n",soma);
    printf("Reajuste ganho: %.2f\n",rej);
    printf("Em percentual: %.0f %\n",percent);

    system ("pause");
    return 0;
}